backend/api_test.py

```diff
@@ -0,0 +1,35 @@
+from fastapi.testclient import TestClient
+from api import app, healthy_response, error_message
+
+client = TestClient(app)
+utterance = "Hello there"
+
+
+def test_health_check_response():
+    response = client.get("/health")
+
+    assert response.status_code == 200
+    assert response.json() == healthy_response
+
+
+def test_chat_success_response(mocker):

```

Comment:
How is this `mocker` injected into the tests?

Reply:
so my understanding is that with `pytest` you can make [fixtures](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html) that are always injected into each test. `pytest-mock` is a plugin that provides the `mocker` as a fixture